solr/core/src/java/org/apache/solr/handler/component/HttpShardHandlerFactory.java

```diff
@@ -267,8 +267,10 @@ public void init(PluginInfo info) {
     }
 
     this.corePoolSize = getParameter(args, INIT_CORE_POOL_SIZE, corePoolSize,sb);
+    this.corePoolSize = Integer.getInteger("remoterequest.threadpool.corepoolsize", this.corePoolSize);
```

Comment:
I assume we want the sys prop option as it's easier than setting `corePoolSize` in `solr.xml? 

For prod, it would be a matter of changing `solrcloud.go` (to modify sys props flag in startup args) in servicetypes vs updating solr config?

Reply:
yeah. in solrcloud.go!